README.rst

```diff
@@ -122,9 +122,40 @@ this is the initial login, the code because this profile is configured
 for use with a passcode device such as a YubiKey. Now when we login
 in to test we are prompted for nothing::
 
-    $ aws --profile prod login
+    % aws --profile prod login
 
 Even if the IdP session has expired in this case we will not be
 prompted for a password because it is stored in the Keyring. The
 user will receive either a phone call or a push to the default
 device configured for Duo to permit authenticating.
+
+Known Issues
+------------
+
+**Unable to authenticate after changing password**
+
+A known problem takes place after changing the underlying password.
```

Comment:
This paragraph seems unnecessary. I would prefer to not mention AD or Illinois. I think this paragraph should be removed, or substantially reduced and rewritten.